src/braket/circuits/gates.py

```diff
@@ -215,6 +215,119 @@ def i(
 Gate.register_gate(I)
 
 
+class GPhase(AngledGate):
+    r"""Global phase gate.
+
+    Unitary matrix:
+
+        .. math:: \mathtt{gphase}(\gamma) = e^(i \gamma) I_1.
+
+    Args:
+        angle (Union[FreeParameterExpression, float]): angle in radians.
+    """
+
+    def __init__(self, angle: Union[FreeParameterExpression, float]):
+        # Avoid parent constructor because _qubit_count must be zero
+        self._qubit_count = self.fixed_qubit_count()
+        self._ascii_symbols = []
+
+        if angle is None:
+            raise ValueError("angle must not be None")
```

Comment:
I don't understand why we have this validation. Angle isn't optional and there's no expectation this method would work without providing angle.

Reply:
Oh I see. Still seems weird to me, but I'm ok with it for consistency